### PR TITLE
Correct extra token, start preparing docker image for TGI/Jetstream Pt

### PR DESCRIPTION
**Pull request by tengomucho**
# What does this PR do?

There was an issue related to the fact that decode used to return the token passed in the decode state as result, making TGI see as if the first token was returned twice. This is resolved by using the data in the decode state (the API should be stable, according to internal discussions with the Jetstream Pytorch team).
Also, the docker image is now capable of serving TGI using Jetstream Pytorch on the supported models.

## Before submitting
- [x] Did you write any new necessary tests?

### CODE REVIEW

File: text-generation-inference/server/text_generation_server/jetstream_pt_support/generator.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
 
 # These will do some bucketing on prefill lengths to avoid too many different sizes
 PREFILL_LENGTHS = [
+    16,
     32,
     64,
```
Comment (mfuntowicz) [inline text-generation-inference/server/text_generation_server/jetstream_pt_support/generator.py:43]: Isnt it a bit too small?

File: text-generation-inference/server/text_generation_server/jetstream_pt_support/token_selector.py [hunk 2]
```diff
@@ -55,6 +55,8 @@ def __init__(
         self.eos_token_ids = eos_token_ids
         self.pad_token_id = pad_token_id
         self.logits_warper = logits_warper
+        # Seed needs to fit a 64-bit integer, so we modulo it in case is bigger (that can happen!)
+        seed = seed % jnp.iinfo(jnp.int64).max
```
Comment (mfuntowicz) [inline text-generation-inference/server/text_generation_server/jetstream_pt_support/token_selector.py:59]: Can't we just store it as int64? Modulo op sounds very expensive

**Review by mfuntowicz — APPROVED** [review]

File: text-generation-inference/server/text_generation_server/jetstream_pt_support/generator.py (hunk 1 above)
Comment (tengomucho) [reply text-generation-inference/server/text_generation_server/jetstream_pt_support/generator.py:43]: umh, it's actually aligned with [Jetstream's](https://github.com/google/JetStream/blob/93de5901a19d5271ceea7de107406b6a40f52c0c/jetstream/engine/token_utils.py#L36) buckets. But I just realized that's been extracted from the function, so I can import the constant now, I'll make the change.

File: text-generation-inference/server/text_generation_server/jetstream_pt_support/token_selector.py (hunk 2 above)
Comment (tengomucho) [reply text-generation-inference/server/text_generation_server/jetstream_pt_support/token_selector.py:59]: The seed comes from the TGI router and sometimes it comes as a crazy big number (bigger than int64). numpy and torch seem to silently do the operation, but `jax` requires the seed to be an int64 and it raises an error if the number is higher. Let me know if you think about a "cheaper" alternative.
